### PR TITLE
EZP-24007: As an editor, I want to see a description of Studio and Studio Plus

### PR DESCRIPTION
**Pull request by yannickroger**
Link: https://jira.ez.no/browse/EZP-24007

## Description
Add presentation pages for "eZ Studio" and "eZ Studio Plus".

studiopresentation

## Tests
Manual and unit tests

## TODO
* [x] Unit test on studio presentation
* [x] Studio plus presentation
* [x] Screenshot or screencast
* [x] Rebase/cleanup commits

### CODE REVIEW

File: Resources/public/js/apps/ez-platformuiapp.js [hunk 1]
```diff
@@ -91,6 +91,10 @@ YUI.add('ez-platformuiapp', function (Y) {
                 type: Y.eZ.DashboardView,
                 parent: 'loginFormView',
             },
+            studioPresentationView: {
+                type: Y.eZ.StudioPresentationView,
+                parent: 'loginFormView',
```
Comment (dpobel) [inline Resources/public/js/apps/ez-platformuiapp.js]: I would not set any parent on this view

File: Resources/public/js/views/services/ez-navigationhubviewservice.js [hunk 2]
```diff
@@ -292,7 +292,14 @@ YUI.add('ez-navigationhubviewservice', function (Y) {
              * @readOnly
              */
             studioNavigationItems: {
-                value: [],
+                valueFn: function () {
+                    return [
+                        this._getNavigationItem(
+                            "eZ Studio presentation", "studio-presentation",
+                            "studioPresentation", {uri: "/studio/presentation"}
```
Comment (dpobel) [inline Resources/public/js/views/services/ez-navigationhubviewservice.js]: the `studioPresentation` route does not expect any parameter so the route params (the last parameter) object should be empty.

File: Resources/public/templates/studiopresentation.hbt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<p>You don't have eZ Studio Installed :(</p>
+
+<h1>Learn more about eZ Studio</h1>
```
Comment (dpobel) [inline Resources/public/templates/studiopresentation.hbt]: usually the title comes first :)

File: Tests/js/views/assets/ez-studiopresentationview-tests.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+                return origTpl.apply(this, arguments);
+            };
+            this.view.render();
+            Y.Assert.isTrue(templateCalled, "The template should have been used to render this.view");
+        },
+    });
+
+    Y.Test.Runner.setName("eZ Studio Presentation View tests");
+    Y.Test.Runner.add(viewTest);
+}, '', {requires: ['test', 'node-event-simulate', 'ez-studiopresentationview']});
```
Comment (dpobel) [inline Tests/js/views/assets/ez-studiopresentationview-tests.js]: the `node-event-simulate` is not needed in this test

**Comment by dpobel** [comment]
seems ok to me, you can apply the same to studio plus presentation.

**Comment by dpobel** [comment]
+1 as soon as Travis agrees :)

**Comment by yannickroger** [comment]
Ready for review if others are interested.

**Comment by andrerom** [comment]
+1, but @rolandbenedetti wanted i frame I think now, so if he creates a url you can use, maybe possible to adopt beginning of next week?

**Comment by rolandbenedetti** [comment]
Agreed with @andrerom 
@yannickroger , I shot you an email about that.
For now you can simply use an iframe that goes to http://ez.no/in-product-studio-teaser
It's an alias, so I will change it to a final landing page later on (for now it goes to the 5.4 web landing page).

Thx

**Comment by yannickroger** [comment]
I am still working on the iframe (70% done). It requires some JS for the positionning. I am still training on this part of YUI.
So I am merging this as is for now and will add the iframe in a follow up.

**Pull request merged**